test/Microsoft.AspNetCore.Blazor.Build.Test/TestFiles/DesignTimeCodeGenerationTest/ChildComponent_WithChildContent/TestComponent.codegen.cs

```diff
@@ -1,34 +1,34 @@
-// <auto-generated/>
-#pragma warning disable 1591
-namespace Test
-{
-    #line hidden
-    using System;
-    using System.Collections.Generic;
-    using System.Linq;
-    using System.Threading.Tasks;
-    public class TestComponent : Microsoft.AspNetCore.Blazor.Components.BlazorComponent
-    {
-        #pragma warning disable 219
-        private void __RazorDirectiveTokenHelpers__() {
-        ((System.Action)(() => {
-global::System.Object __typeHelper = "*, TestAssembly";
-        }
-        ))();
-        }
-        #pragma warning restore 219
-        #pragma warning disable 0414
-        private static System.Object __o = null;
-        #pragma warning restore 0414
-        #pragma warning disable 1998
-        protected override void BuildRenderTree(Microsoft.AspNetCore.Blazor.RenderTree.RenderTreeBuilder builder)
-        {
-            base.BuildRenderTree(builder);
-            builder.AddAttribute(-1, "ChildContent", (Microsoft.AspNetCore.Blazor.RenderFragment)((builder2) => {
-            }
-            ));
-        }
-        #pragma warning restore 1998
-    }
-}
-#pragma warning restore 1591
+// <auto-generated/>

```

Comment:
My previous attempt to fix the line ending issues on Mac seems to have some bugs. I hope this is fixed it ...